### PR TITLE
chore: header use d2tech

### PR DESCRIPTION
**Pull request by Zaynex**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |    头部增加第十八届D2终端大会推广       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 654ea36</samp>

Add a banner for D2 conference on the website header. Update `locales`, image, link, and GA event label in `./.dumi/theme/slots/Header/index.tsx`.

### <samp>🤖 Generated by Copilot at 654ea36</samp>

* Update the banner message and link for Chinese users to promote the 18th D2 Frontend Technology Conference ([link](https://github.com/ant-design/ant-design/pull/45756/files?diff=unified&w=0#diff-2b1224dbeb7d20f1cb1b4590eb12b68104c605d2a67f0b0bd4fd9d8b0100a728L28-R31))
* Remove the Yuque logo and use the `locale.link` property for the banner link and the Google Analytics event label ([link](https://github.com/ant-design/ant-design/pull/45756/files?diff=unified&w=0#diff-2b1224dbeb7d20f1cb1b4590eb12b68104c605d2a67f0b0bd4fd9d8b0100a728L374-R382), [link](https://github.com/ant-design/ant-design/pull/45756/files?diff=unified&w=0#diff-2b1224dbeb7d20f1cb1b4590eb12b68104c605d2a67f0b0bd4fd9d8b0100a728L388-R388))

### CODE REVIEW

File: .dumi/theme/slots/Header/index.tsx [hunk 1]
```diff
@@ -371,21 +371,21 @@ const Header: React.FC = () => {
             className={styles.banner}
             message={
               <>
-                <img
+                {/* <img
                   className={styles.icon}
                   src="https://gw.alipayobjects.com/zos/rmsportal/XuVpGqBFxXplzvLjJBZB.svg"
                   alt="yuque"
-                />
+                /> */}
```
Comment (afc163) [inline .dumi/theme/slots/Header/index.tsx:378]: 删掉就好

Comment (Zaynex) [reply .dumi/theme/slots/Header/index.tsx]: 好了

**Comment by MadCcc** [comment]
Banner 底色换一下吧

**Comment by Zaynex** [comment]
> Banner 底色换一下吧

需要改成什么颜色呢？有推荐的吗

**Comment by Zaynex** [comment]
image

@MadCcc  换了跟D2官网配色接近的颜色。

File: .dumi/theme/slots/Header/index.tsx [hunk 2]
```diff
@@ -25,10 +25,10 @@ const RESPONSIVE_SM = 1200;
 
 const locales = {
   cn: {
-    message:
-      '语雀公益计划：大学生认证教育邮箱，即可免费获得语雀会员。语雀，支付宝匠心打造的在线文档平台。',
-    shortMessage: '支付宝语雀 · 大学生公益计划火热进行中！',
-    more: '了解更多',
+    message: '第十八届D2终端技术大会火热来袭，以技术会友、一起分享技术的乐趣。',
+    shortMessage: '第十八届D2终端技术大会火热来袭！',
```
Comment (afc163) [inline .dumi/theme/slots/Header/index.tsx:29]:
```suggestion
    message: '第十八届 D2 终端技术大会火热来袭，以技术会友，一起分享技术的乐趣。',
    shortMessage: '第十八届 D2 终端技术大会火热来袭！',
```

**Review by afc163 — APPROVED** [review]

**Pull request merged**